import-service/serverless.ts

```diff
@@ -85,6 +101,36 @@ const serverlessConfiguration: Serverless = {
       ],
     },
   },
+  resources: {
+    Resources: {
+      GatewayResponseAccessDenied: {
+        Type: 'AWS::ApiGateway::GatewayResponse',
+        Properties: {
+          ResponseParameters: {
+            'gatewayresponse.header.Access-Control-Allow-Origin': "'*'",
+            'gatewayresponse.header.Access-Control-Allow-Headers': "'*'",
```

Comment:
Try to change **gatewayresponse.header.Access-Control-Allow-Headers** by **gatewayresponse.header.Access-Control-Allow-Credentials**

Reply:
it handled this way:
https://github.com/skyRoma/nodejs-aws-be/pull/5/commits/2b376de08eed6f945c347e76d1f1c0ba33e691d8